src/components/NewMessage/NewMessageAbsenceInfo.vue

```diff
@@ -122,16 +123,16 @@ export default {
 		},
 	},
 
-	watch: {
-		userAbsenceMessage() {
-			this.$nextTick(() => {
-				this.setIsTextMoreThanOneLine()
-			})
-		},
-	},
-
 	mounted() {
 		this.setIsTextMoreThanOneLine()
+		this.resizeObserver = new ResizeObserver(this.setIsTextMoreThanOneLine)
+		this.resizeObserver.observe(this.$refs.absenceMessage)
+	},
+
+	beforeDestroy() {
+		if (this.resizeObserver) {
+			this.resizeObserver.disconnect()
+		}
```

Comment:
you don't have conditions for starting observer, so no need to check when destroying?
```suggestion
		this.resizeObserver.disconnect()
```

Reply:
Or use `useResizeObserver` from vueuse :D

Reply:
It's not used anywhere in project yet.. but we can keep it in mind for ts migration in each case

Reply:
TS?

Reply:
I meant `script setup` 👀. But we do it together usually

Reply:
We can use composables in Options API, even mixing with properties from Options API.